tests/contracts/txParserBTCTest.ts

```diff
@@ -14,7 +14,7 @@ import {
     FixedArray,
 } from 'scrypt-ts'
 
-export class TransactionParserTest extends SmartContract {
+export class TransactionParserTestBTC extends SmartContract {
```

Comment:
I don't think this should be part of this PR. Revert please.

Reply:
There is another class with same name that is why the CI is failling, that is the reason i rename it